README.md

```diff
@@ -102,6 +102,38 @@ allow openvpn_t openvpn_etc_t:file { write unlink };
 end
 ```
 
+### fcontext
+
+Allows managing the SELinux context of files. This may be needed to allow certain daemons access to their files and similar.
```

Comment:
Please rephrase:
```text
Allows managing the SELinux context of files.  
This can be used to grant SELinux-protected daemons access to additional / moved files
```